core/src/main/java/org/jboss/hal/core/modelbrowser/ModelBrowser.java

```diff
@@ -169,6 +170,10 @@ public ModelBrowser(CrudOperations crud,
                         .on(click, event -> refresh(tree.getSelected()))
                         .title(resources.constants().refresh())
                         .add(Elements.i().css(fontAwesome(CSS.refresh))).element())
+                .add(expandAll = button().css(btn, btnDefault)
```

Comment:
Please remove the expand all feature from the model browser. The tree is too large. The page crashes, when I click on expand all on my machine.

Reply:
I am sorry about that. I removed it.

Reply:
No worries. Besides that: Good job!
Thanks for your contribution @khermano

Reply:
Thank you. :)